Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -72,8 +72,13 @@ public AwsUserProfilesPage(VisualElement container, StateManager stateManager)
                 
             RefreshProfiles();
 
-            container.Q<DropdownField>("UserProfilePageAccountNewProfileRegionDropdown").choices =
-                _stateManager.CoreApi.ListAvailableRegions().ToList();
+            var regions = _stateManager.CoreApi.ListAvailableRegions().ToList();
+            var filteredList = regions
+                .Where(item => !item.Contains("cn-"))
```

Comment:
Would it be cleaner to just remove `cn-north-1` and `cn-northwest-1` from the AwsRegionMapper list instead?